core/src/main/java/nl/nn/adapterframework/filesystem/ConnectedFileSystemBase.java

```diff
@@ -85,19 +88,19 @@ public void close() throws FileSystemException {
 			}
 		}
 	}
-	
+
 
 	/**
 	 * Get a Connection from the pool, or the global shared connection.
 	 */
 	protected C getConnection() throws FileSystemException {
 		try {
-			return isPooledConnection() ? connectionPool.borrowObject() : globalConnection;
+			return isPooledConnection() ? connectionPool!=null ? connectionPool.borrowObject() : null : globalConnection;
```

Comment:
Klopt dit? Mag deze methode null teruggeven? Ik zou een fout verwachten als er geen pool is terwijl hij op isPooling staat.